### PR TITLE
Feature/offramp page

### PR DESCRIPTION
**Pull request by katedee**
### TL;DR:
- added offramp page
- added a banner include/mixin for some banners in the app
- updated "no" on postResidence to point to the offramp page
- just a basic test that the page loads

Offramp is in its own directory, because I can see us using this outside of just the personal controller. If you feel that it does not belong in its own folder, we can change this.

Still calling it offramp, this can change depending on what we decide to name it.

Banner include/mixin - we only have the blue one on the app right now (and it needs to be added to a bunch of places), but I allowed for colour variant in case it does change up.

image

### CODE REVIEW

File: routes/personal/personal.controller.js [hunk 1]
```diff
@@ -59,8 +59,7 @@ const postMaritalStatus = (req, res) => {
 
 const postResidence = (req, res) => {
   if (req.body.residence !== 'Yes') {
-    /* TODO: Point this at the offboarding page */
-    return res.redirect('/start')
+    return res.redirect('/offramp')
```
Comment (pcraig3) [inline routes/personal/personal.controller.js]: Let's update the test for the "residence" page to show that it redirects here.

File: public/scss/components/_banners.scss [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    &--blue {
+        background: $color-banner-blue-light;
+        border-left: 5px solid $color-banner-blue;
+
+        &__icon {
+            background: $color-banner-blue;
+        }
+    }
+}
```
Comment (pcraig3) [inline public/scss/components/_banners.scss]: The notification banner looks super! Good work!

File: views/offramp/offramp.pug [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+block content
+
+  h1 #{title}
+  +blueBanner('Your tax return is too complex to be filed using this service. Nothing about your taxes is incorrect – see below for further instructions on how to complete your tax return.', 'blue')
+  div
+    p To finish filing your taxes for the current tax year (filing for 2018), you will need to do one of the following:
+    ul
+      li Use the paper form enclosed in the Claim Tax Benefits package you received in the mail
+      li Contact CRA at (123) 456-7890 for help
+      li File your taxes with another free service, listed here: #[a(href="#") Ways to file your taxes]
```
Comment (pcraig3) [inline views/offramp/offramp.pug]: A few comments about this:

- Let's add a new link: "[Visit a community tax clinic near you](https://www.canada.ca/en/revenue-agency/services/tax/individuals/community-volunteer-income-tax-program.html) to have a volunteer help you submit your taxes."
- Let's make the phone number a "tel" link: https://css-tricks.com/the-current-state-of-telephone-links/
- For the last point, let's say: "File your taxes with another [NETFILE-compliant online service certified by the CRA](https://www.canada.ca/en/revenue-agency/services/e-services/e-services-individuals/netfile-overview/certified-software-netfile-program.html)"
